<ui version="4.0" >
 <class>DlgExam</class>
 <widget class="QDialog" name="DlgExam" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Pr√ºfungssimulation</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="labHeader" >
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Bezeichnung</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Frage</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QTextBrowser" name="textBrowser" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Ihre Antwort</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rbA" >
            <property name="text" >
             <string>&amp;A</string>
            </property>
            <property name="shortcut" >
             <string>A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbB" >
            <property name="text" >
             <string>&amp;B</string>
            </property>
            <property name="shortcut" >
             <string>B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbC" >
            <property name="text" >
             <string>&amp;C</string>
            </property>
            <property name="shortcut" >
             <string>C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbD" >
            <property name="text" >
             <string>&amp;D</string>
            </property>
            <property name="shortcut" >
             <string>D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbNoIdea" >
            <property name="text" >
             <string>&amp;Keine Ahnung</string>
            </property>
            <property name="shortcut" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Navigation</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbFirst" >
            <property name="maximumSize" >
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbPrev" >
            <property name="maximumSize" >
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox" />
          </item>
          <item>
           <widget class="QPushButton" name="pbNext" >
            <property name="maximumSize" >
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>></string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbLast" >
            <property name="maximumSize" >
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>>></string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="maximumSize" >
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title" >
          <string>Zeit</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="labProgessTime" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="pgTime" >
            <property name="value" >
             <number>24</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="maximumSize" >
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title" >
          <string>Beantwortete Fragen</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="labProgressQuestions" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="pgQuestions" >
            <property name="value" >
             <number>24</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6" >
         <property name="maximumSize" >
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title" >
          <string>Auswertung</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Richtige Antworten:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Falsche Antworten:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="labWrong" >
            <property name="text" >
             <string>?</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="labCorrect" >
            <property name="text" >
             <string>?</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>max. erlaubte falsche Antworten:</string>
            </property>
            <property name="indent" >
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLabel" name="labMaxWrong" >
            <property name="text" >
             <string>10</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labResult" >
         <property name="font" >
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbFinish" >
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Abgeben &amp;&amp; auswerten</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbCancel" >
           <property name="minimumSize" >
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Abbrechen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgExam</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>468</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel" >
     <x>514</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
